app/uk/gov/hmrc/trackingconsentfrontend/testonly/views/TestPage.scala.html

```diff
@@ -22,7 +22,7 @@
 )
 @()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@defining(Messages("testPage.title"), Messages("testPage.heading")) { case (title, heading) =>
+@defining((Messages("testPage.title"), Messages("testPage.heading"))) { case (title, heading) =>
```

Comment:
it did work before, but this is an example of where I think before we silenced all warnings from templates and now we're just silencing unused imports so this surfaced - it wanted an explicit tuple